Solutions/Silverfort/Data Connectors/SilverfortAma.json

```diff
@@ -0,0 +1,132 @@
+{
+    "id": "SilverfortAma",
+    "title": "Silverfort Admin Console",
+    "publisher": "Silverfort",
+    "descriptionMarkdown": "The [Silverfort](https://silverfort.com) ITDR Admin Console connector solution allows ingestion of Silverfort events and logging into Microsoft Sentinel.\n Silverfort provides syslog based events and logging using Common Event Format (CEF). By forwarding your Silverfort ITDR Admin Console CEF data into Microsoft Sentinel, you can take advantage of Sentinels's search & correlation, alerting, and threat intelligence enrichment on Silverfort data. \n Please contact Silverfort or consult the Silverfort documentation for more information.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "DATATYPE_NAME",

```

Comment:
Please replace DATA_TYPE name with data type name. 
You can refer to this Connector for more clarification 
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/OneIdentity/Data%20Connectors/OneIdentity.JSON

Reply:
Resolved